src/Components/Contributors/AffiliationDisplay.tsx

```diff
@@ -67,8 +80,8 @@ const AffiliationDisplay: FC<AffiliationDisplayProps> = ({
   return (
     <StyledAffiliationsWrapper backgroundcolor={backgroundcolor} variant="outlined" data-testid={dataTestid}>
       <StyledCardContent>
-        <Grid container justifyContent="space-between" spacing={2}>
-          <Grid item xs={6}>
+        <Grid container justifyContent="space-between" spacing={1}>
+          <Grid item xs={true}>
```

Comment:
tror det holder å bare skrive det slik
```suggestion
          <Grid item xs>
```

Reply:
Fikset